index/store/moss/lower.go

```diff
@@ -105,8 +105,9 @@ type llSnapshot struct {
 	llStore  *llStore // Holds 1 refs on the llStore.
 	kvReader store.KVReader
 
-	m    sync.Mutex // Protects fields that follow.
-	refs int
+	m              sync.Mutex // Protects fields that follow.
+	refs           int
+	childSnapshots map[string]*llSnapshot
```

Comment:
It looks lik childSnapshots is in the same paragraph (not separated by newline) of the section with mutex labeled "protects fields that follow", but the operations that use this field do not acquire the mutex.